lib/model/CoarseChannel.mjs

```diff
@@ -38,7 +38,7 @@ const channelTypeConstraints = {
   'Fog': [`Fog`, `FogOutput`, `FogType`],
   'Speed': [`StrobeSpeed`, `StrobeDuration`, `PanTiltSpeed`, `EffectSpeed`, `EffectDuration`, `PrismRotation`, `Rotation`, `Speed`, `Time`, `WheelSlotRotation`, `WheelRotation`, `WheelShake`],
   'Maintenance': [`Maintenance`],
-  'Intensity': [`Intensity`, `BeamAngle`, `Generic`],
+  'Intensity': [`Intensity`, `BeamAngle`, `BeamPosition`, `Generic`],
```

Comment:
`BeamAngle` and `BeamPosition` should both not be `Intensity` channels (maybe rather `Rotation`?), see the fixture export diff.